Dockerfile.tests

```diff
@@ -0,0 +1,11 @@
+FROM fedora:30
```

Comment:
`FROM fedora:rawhide` if possible, so that we do not have to change it every EOL release